src/ui/layout/mct-tree.vue

```diff
@@ -261,13 +273,20 @@ export default {
 
         if (root.identifier !== undefined) {
             let rootNode = this.buildTreeItem(root);
+            let rootComposition = this.openmct.composition.get(root);
```

Comment:
Naming is slightly confusing here. `rootComposition` refers to a `CompositionCollection` object, and `compositionCollection` references an Array. I'd be inclined to name the first one `compositionCollection`, and the array `rootComposition`.

Reply:
Yep, good call. Renamed.